scripts/check_for_blobs.sh

```diff
@@ -1,19 +1,30 @@
 #!/bin/bash
 
-blobs=$(find . -type f -exec sh -c '
-  for f; do 
-    mimetype -b "$f" | grep -Eq "application/(x-object|x-executable)" && 
-    printf "%s\n" "$f"
+declare -a blobs
+
+KNOWN_GOOD_FILE_EXTENSIONS=("rs" "c" "h" "cc" "sh" "py" "toml" "yml" "json" "md" "gitignore" "png")
+
+while read -r file; do
+  # NOTE: mimetype detection spawns a perl process for each file and is pretty slow.
+  # we work around this by skipping files with known-good extensions.
+  ext="${file##*.}"
+  for skipExt in "${KNOWN_GOOD_FILE_EXTENSIONS[@]}"; do
```

Comment:
I don't want to be advocating _for_ regex, but probably this could have been done faster with a regex :P